lgtm/__init__.py

```diff
@@ -3,7 +3,7 @@
 import integrations
 
 
-def pull_request_ready_to_merge(github_token, org, repo, pr_number, owners_file='OWNERS'):
+def pull_request_ready_to_merge(github_token, org, repo, pr_number, owners_file='OWNERS', options={}):
```

Comment:
Can't have a default that's a mutable. 

http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments

I suggest we use `**options`, which will default to an empty dict.